### PR TITLE
Update autoinstallation copy for containerized server

### PR DESCRIPTION
**Pull request by aaannz**
# Description

This PR updates instruction how to register autoinstallation distribution with containerized server.

# Target branches

Uyuni, SUMA5

Backport targets (edit as needed):

- master

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/24111
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 1]
```diff
@@ -4,6 +4,8 @@
 The autoinstallation process relies on an several files to initiate the installation.
 These files include the Linux kernel, an initial RAM disk, and other files required to boot the operating system in installation mode.
 
+{productname} use [systemitem]``mgradm`` tool to copy installation file from the source to the server container.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:7]:
```suggestion
{productname} uses the [systemitem]``mgradm`` tool to copy the installation file from the source to the server container.
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 2]
```diff
@@ -28,21 +30,25 @@ This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [p
 
 
 .Procedure: Importing Files from Installation Media
-. Copy the installation media to your {productname} Server.
-  For {suse} operating systems, you can download installation media from https://www.suse.com/download/.
-. Loop-mount the ISO image, and copy its contents somewhere:
+. Use [systemitem]``mgradm`` tool to import installation data from the ISO
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:33]:
```suggestion
. Use [command]``mgradm`` to import installation data from the ISO:
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 3]
```diff
@@ -28,21 +30,25 @@ This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [p
...
 ----
-# mount -o loop,ro <image_name>.iso /mnt
-# mkdir -p /srv/www/distributions
-# cp -a /mnt /srv/www/distributions/<image_name>
-# umount /mnt
+# mgradm distribution copy <image_name>.iso <image_name>
 ----
 +
+. Take a note of the distribution path reported by [systemitem]``mgradm`` tool
+  You will need it when you declare the distribution to {productname}.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:40]:
```suggestion
. Take a note of the distribution path reported by [command]``mgradm``.
  You will need it when you declare the distribution to {productname}.
  
  
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 4]
```diff
@@ -28,21 +30,25 @@ This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [p
...
+
+=== Distribution autodetection and registration
 
-. Take a note of the file path.
-You will need it when you declare the distribution to {productname}.
+[systemitem]``mgradm`` is able to automatically detect distribution name and register it to the server. Provided ISO image needs to contain [path]``.treeinfo`` file.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:44]:
```suggestion
[systemitem]``mgradm`` is able to automatically detect distribution name and register it to the server.
Provided ISO image needs to contain a [path]``.treeinfo`` file.
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 5]
```diff
@@ -28,21 +30,25 @@ This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [p
...
 
+.Procedure: Import distribution file with autodetection and registration
+. Use [systemitem]``mgradm`` tool
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:47]:
```suggestion
. Use [command]``mgradm``:
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 6]
```diff
@@ -56,9 +62,21 @@ It is simpler than importing contents from an installation media, because it use
...
-. Determine where the installation files went with the command [command]``ls -d /usr/share/tftpboot-installation/*``.
-  Take a note of the file path.
+. You can install the package to different root to avoid need for restart using following command:
++
+----
+# mkdir /opt/tftpinstall
+# zypper --installroot /opt/tftpinstall install tftpboot-installation-SLE-Micro-5.5-x86_64
+----
+. Determine where the installation files went with the command [command]``ls -d /opt/tftpinstall/usr/share/tftpboot-installation/*``.
+. Copy installation files using [systemitem]``mgradm`` tool
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:72]:
```suggestion
. Find the installation files with the command [command]``ls -d /opt/tftpinstall/usr/share/tftpboot-installation/*``.
. Copy installation files using [command]``mgradm``:
```

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 7]
```diff
@@ -56,9 +62,21 @@ It is simpler than importing contents from an installation media, because it use
...
++
+----
+# mgradm distribution copy /opt/tftpinstall/usr/share/tftpboot-installation/SLE-Micro-5.5-x86_64 SLE-Micro-5.5-x86_64
+----
+. Take a note of the distribution path reported by [systemitem]``mgradm`` tool
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:77]:
```suggestion
. Take a note of the distribution path reported by [systemitem]``mgradm`` tool.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
cosmetic suggestions

**Review by 0rnela — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/autoinst-distributions.adoc [hunk 8]
```diff
@@ -28,21 +30,28 @@ This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [p
 
 
 .Procedure: Importing Files from Installation Media
-. Copy the installation media to your {productname} Server.
-  For {suse} operating systems, you can download installation media from https://www.suse.com/download/.
-. Loop-mount the ISO image, and copy its contents somewhere:
+. Use [command]``mgradm`` to import installation data from the ISO:
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-distributions.adoc:33]:
```suggestion
. Use [command]``mgradm`` to import installation data from the ISO image:
```

**Review by keichwa — APPROVED** [review]
lgtm

**Comment by keichwa** [comment]
@aaannz if you do not veto, we'd merge it for you.

**Comment by aaannz** [comment]
I don't feel like giving my PR a review :) so LGTM and go ahead. I can't merge it myself.

**Pull request merged**